src/main/java/teammates/common/datatransfer/questions/FeedbackRankQuestionDetails.java

```diff
@@ -62,15 +62,13 @@ public abstract String getQuestionWithoutExistingResponseSubmissionFormHtml(
      * Updates the mapping of ranks for the option optionReceivingPoints.
      */
     protected void updateOptionRanksMapping(
-                        Map<String, List<Integer>> optionRanks,
-                        String optionReceivingRanks, int rankReceived) {
-        if (!optionRanks.containsKey(optionReceivingRanks)) {
-            List<Integer> ranks = new ArrayList<>();
-            optionRanks.put(optionReceivingRanks, ranks);
-        }
+            Map<String, List<Integer>> optionRanks,
+            String optionReceivingRanks, int rankReceived) {
 
+        optionRanks.putIfAbsent(optionReceivingRanks, new ArrayList<>());
```

Comment:
Can use `computeIfAbsent`?

Reply:
Hi thx for the review! Regarding this part of code, i ve posted my question here:
https://github.com/TEAMMATES/teammates/pull/8226/files/cc08ccb5bdf286b89c0f0753633a8281a6196d2f#diff-d24b10bd0f20178eaa5805ec5db45760